Desktop version

RileyManda opened the pull request:
I have completed work on the Desktop Version of the Portfolio Project according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md).
Here are the main features I have added to the desktop version: 
* Used media queries to ensure the website displays correctly on desktop screens [(min-width: 768px)](https://github.com/RileyManda/portfolio-website/blob/desktop-version/style.css)
* Made sure the components look great and function well on different devices.
* Modified header, about, works, and contact section with additional features:

-       Added desktop header .about and contact section background images.
-       Aligned the title texts for desktop screen sizes
-       Used grid layout and flex layout to align elements.
-       Made sure that there are no linter errors
         
* In addition I Made sure that :

- There are no linter errors in the project.
- Used Correct git-flow.
- Documented my work in a professional manner according to requirements,[(https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).].
- Included a descriptive [Readme](https://github.com/RileyManda/portfolio-website/blob/desktop-version/README.md) file.
- Used descriptive commit messages
-  Followed the recommended HTML&CSS best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Please review the changes and let me know if there are any issues or suggestions for improvement. I look forward to your feedback. 
Thank you! 
Best regards,🙏

V-Blaze commented on index.html:
```diff
@@ -128,14 +135,12 @@ <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
-                        feature or website Look through <br>
-                        some of my work and experience! If<br>
-                        you like what you see and have a <br>
-                        project you need coded,<br>
-                        don’t hesitate to contact me.
+                    <p class="supporting-text">
+                        I can help you build a product, feature or website. Look through some of my work and <br>
+                        experience! If you like what you see and have <br>a project you need coded, <br>don't hesitate
+                        to contact me.
```
- [ ] You have done a great job here. :+1: :+1:, But you still need to make some improvements before moving to the next milestone. :smile: 
- [ ] As part of the requirement for the desktop version is no make your design look like the Figma design as much as possible, I think your tex placement doesn't match the one in the Figma design. I'll suggest you look into this and align your text properly to the center. :+1: :+1: 
- - Your design:
![Screenshot from 2023-05-11 23-16-02](https://github.com/RileyManda/portfolio-website/assets/107632780/1ef9a208-c5d8-4551-b32e-656377a38381)

- - Figma design:
![Screenshot from 2023-05-11 23-16-22](https://github.com/RileyManda/portfolio-website/assets/107632780/6c0223aa-1c82-4961-8347-a46757843864)

V-Blaze commented on index.html:
```diff
@@ -128,14 +135,12 @@ <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
+                        I can help you build a product, feature or website. Look through some of my work and <br>
+                        experience! If you like what you see and have <br>a project you need coded, <br>don't hesitate
+                        to contact me.
                     </p>
                     <button class="project-btn">Get my resume</button>
                     <div class="about-img"></div>
```
- [ ] Also in this section, Looking at the `Skill, Framwork` lists, according to the given design, they should be aligned to the center, I'll suggest you take a look at this also and update your design. :smile: :+1: :100: 

- - Figma design:
![Screenshot from 2023-05-11 23-16-55](https://github.com/RileyManda/portfolio-website/assets/107632780/99cfea28-37f8-45ed-89c5-74b374bc01de)

- - Your current design:
![Screenshot from 2023-05-11 23-17-15](https://github.com/RileyManda/portfolio-website/assets/107632780/169ef286-d59a-45c7-936b-aa94f9fb2980)

V-Blaze commented on index.html:
```diff
@@ -200,58 +205,75 @@ <h3 class="about-card-title">Skills</h3>
-                        <input type="submit" value="Get in touch" class="contact-btn">
-                    </div>
+            <div class="contact-container">
+                <div class="grid-container">
+                    <h2 class="contact-text grid-item">I'm always interested in hearing about new projects, so if you'dlike to chat please get in touch.</h2>
+                    <div class="form-container grid-item">
+                        <form class="contact-form" action="https://formspree.io/f/mvonlebo" method="POST">
+                            <div class="desktop-fields">
+                                <div class="form-group">
+                                    <label for="firstname"></label>
+                                    <input type="text" id="first-name" name="first-name" placeholder="First Name" maxlength="30" required>
+                                </div>
+                                <div class="form-group">
+                                    <label for="lastname"></label>
+                                    <input type="text" id="last-name" name="last-name" placeholder="Last Name" maxlength="30" required>
+                                </div>
```
- [ ] I also think you could make some adjustments to your design here. :+1:  Looking at he screenshots below, you can see that your `first name` and `last name` input are not properly stretched and aligned like the Figma design, I'll suggest you update the styling here. :+1: 
 - [ ] Also the button is positioned at the left and not at the center, following the Figma design, I'll suggest you update your design accordingly as being able to replicate a given design is a very valuable skill for a front-end developer. :smile: :wave: 

- - Your design:
![Selection_038](https://github.com/RileyManda/portfolio-website/assets/107632780/261034cb-e210-4aee-8090-baf36b027c8d)

- - Figma design:
![Screenshot from 2023-05-11 23-17-42](https://github.com/RileyManda/portfolio-website/assets/107632780/59df57bb-0fc7-40ee-9d5e-f6967dca5f3c)

RileyManda replied:
I have completed iterations on the Desktop Version of the Portfolio Project as requested by the reviewer.
Here are the iterations i have made to the project: 

- Replaced first name and last name form fields with full name field as this is optional according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md): 
- Fixed the form button alignment-positioning it to the left.
- Centered the card tags.
- Updated the About Section and aligned text to centered to match the UX Design.
- Aligned cards to center in desktop view
In addition: 

- I have aligned the card container to center for desktop view.

- Added nth element CSS to re-arrange social icons to match desktop view._**(Mobile and Desktop icons alignment is different according to the design).**_

In general, The project has been designed to meet the following requirements:

- Used media queries to ensure the website displays correctly on desktop screens [(min-width: 768px)](https://github.com/RileyManda/portfolio-website/blob/desktop-version/style.css)
-   Made sure the components look great and function well on different devices.
-   Used grid layout and flex layout to align elements.
-   Made sure that there are no linter errors
-   Attempted to replicate the UX design as much as possible.
-  Ensured there are no linter errors in the project.
-  Used Correct git-flow.
- Documented my work in a professional manner according to requirements,
[(https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).].
- Included a descriptive [Readme](https://github.com/RileyManda/portfolio-website/blob/desktop-version/README.md) file.
- Used descriptive commit messages
- Followed the recommended HTML&CSS best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Please review the changes and let me know if there are any issues or suggestions for improvement. I look forward to your feedback. 

Thank you! 
Best regards,🙏

V-Blaze reviewed (changes requested):
## Hi @RileyManda  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![giphy](https://user-images.githubusercontent.com/107632780/229314554-3700de13-2c00-46d5-bc77-46ad09af7467.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
-  Very detailed and descriptive pull request summary👋 
-  Readme looks well polished and proffesional✔ :heavy_check_mark: :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

RileyManda commented:
I have completed iterations on the Desktop Version of the Portfolio Project as requested by the reviewer.
Here are the iterations i have made to the project: 

Replaced first name and last name form fields with full name field as this is optional according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md):

Fixed the form button alignment-positioning it to the left.

Centered the card tags.

Updated the About Section and aligned text to centered to match the UX Design.

Aligned cards to center in desktop view
In addition:

I have aligned the card container to center for desktop view.

Added nth element CSS to re-arrange social icons to match desktop view.(Mobile and Desktop icons alignment is different according to the design).

In general, The project has been designed to meet the following requirements:

Used media queries to ensure the website displays correctly on desktop screens [(min-width: 768px)](https://github.com/RileyManda/portfolio-website/blob/desktop-version/style.css)
Made sure the components look great and function well on different devices.
Used grid layout and flex layout to align elements.
Made sure that there are no linter errors
Attempted to replicate the UX design as much as possible.
Ensured there are no linter errors in the project.
Used Correct git-flow.
Documented my work in a professional manner according to requirements,
[(https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).].
Included a descriptive [Readme](https://github.com/RileyManda/portfolio-website/blob/desktop-version/README.md) file.
Used descriptive commit messages
Followed the recommended HTML&CSS best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)
Please review the changes and let me know if there are any issues or suggestions for improvement. I look forward to your feedback. 

Thank you! 
Best regards,🙏

KanzaTahreem reviewed (approved):
# Approved 🥳

Hi @RileyManda 🙋‍♀️

Great job implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**

<img src="https://media4.giphy.com/media/cFkiFMDg3iFoI/giphy.gif"/>

## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Good branch name. Great job!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_**Please see the comments under review**_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
